org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/component/Tab.java

```diff
@@ -155,6 +156,12 @@ public static DecorateInfo get (MInfoWindow mInfo){
 			}
 			return null;
 		}
+
+		public static DecorateInfo get(MForm form){
+			if (form != null)
```

Comment:
should be form != null && !Util.isEmpty(form.getImageURL(), true)

Reply:
indeed, done